configs/components/pe-bolt-services.rb

```diff
@@ -55,7 +55,4 @@
     "/bin/chown -R #{settings[:pe_bolt_user]}:#{settings[:pe_bolt_user]} #{settings[:plan_logdir]}",
     "/bin/chown -R #{settings[:pe_bolt_user]}:#{settings[:pe_bolt_user]} #{settings[:plan_rundir]}"
   ]
-  pkg.add_postinstall_action ["install"], [
-    "systemctl daemon-reload"
```

Comment:
This wasn't needed for systemd?

Reply:
I'd assume we wouldn't see an issue until the systemd config changed.

Reply:
I think you're right - I only verified that installing it worked as expected, since I thought the daemon-reload was to load a newly installed service rather than update service config. I'll verify this is an issue later this morning and put up a PR to add the postinstall to systemd platforms if it is.